Sources/xcprojectlint-package/MissingTargetReferencesInScheme.swift

```diff
@@ -0,0 +1,54 @@
+/*
+* Copyright (c) 2018 American Express Travel Related Services Company, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software distributed under the License
+* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+* or implied. See the License for the specific language governing permissions and limitations under
+* the License.
+*/
+
+import Foundation
+import XcodeProj
+
+public func missingTargetReferencesInSchemes(workspace: XCWorkspace) -> Report {
+  func getAllFiles(in element: XCWorkspaceDataElement) -> [XCWorkspaceDataFileRef] {
```

Comment:
how come this is a nested function and not just a private function like `getProjectsReferenced` ?

Reply:
No reason, just drafted this way. I'll make it a private func if we decide to move further with this check.